src/core/core.cpp

```diff
@@ -71,13 +73,21 @@ void Stop() {
 
 /// Initialize the core
 void Init() {
+#ifdef ARCHITECTURE_x86_64
     if (Settings::values.use_cpu_jit) {
         g_sys_core = std::make_unique<ARM_Dynarmic>(USER32MODE);
         g_app_core = std::make_unique<ARM_Dynarmic>(USER32MODE);
     } else {
         g_sys_core = std::make_unique<ARM_DynCom>(USER32MODE);
         g_app_core = std::make_unique<ARM_DynCom>(USER32MODE);
     }
+#else
+    if (Settings::values.use_cpu_jit) {
+        LOG_WARNING(Core, "Using CPU Interpreter. CPU JIT is activated in config but not available on current platform!");
+    }
```

Comment:
This warning should be moved to where settings are applied so it can be used for other features (like the shader JIT) too.
Reasoning: We can't do this kind of check for shader JIT because that will try to create a lot of shaders and would spam every time a new one is added.